model_analyzer/constants.py

```diff
@@ -57,5 +57,8 @@
 # Logging
 LOGGER_NAME = "model_analyzer_logger"
 
+# PA Error Log Filename
+PA_ERROR_LOG_FILENAME = "perf_analyzer_error.log"
```

Comment:
should this be configurable? Is there any chance the CWD isn't writable?
What if the file already exists from a previous run? Is this deleted?  Was any thought given to using a temp file?

Reply:
By default we write to CWD, but the user could change this by specifying an export path. I'll change this so that the error log file also goes to the export path.

Reply:
export_path instead of perf_output_path?

Reply:
Yes, there will not be a perf output path specified in most cases (since perf logging is off by default).

Reply:
And, it seemed like a lot of unneeded code to figure out when to switch btw. using export vs. perf_output path.

Reply:
no switching between. It would always be perf_output_path. If unspecified, it defaults to CWD. It doesn't make sense that perf_output_path controls half of the potential PA output, and that the other goes into a place where all of our data goes.

Reply:
```If perf_output_path is None, output will be written to stdout.```
Got it, ok that doesn't default like I had expected.  export path it is 👍 

Are there any doc changes needed about this new output file and what controls where it goes?

Reply:
We append it to the end of a previous log file, but will only show the warning message if new errors show up.

Reply:
Yeah I think we need to delete it. It would be too confusing otherwise.
Imagine they have some shape errors. They look at the error log and go fix it. Now they rerun and there is a new PA failure. They open the file and see (no longer valid) shape errors.

Reply:
Done